client/src/app/+about/about.component.html

```diff
@@ -1,3 +1,12 @@
-<p>
-  about works!
-</p>
+
+    <h3> אודות</h3>
+    <p>נכון לסוף 2013, כטריליון שקלים מנוהלים בקרנות הפנסיה וקופות הגמל בישראל.</p>כספים אלו - המיועדים להבטחת רמת חיינו לאחר גיל הפרישה לפנסיה - מנוהלים ע"י הגופים המוסדיים בישראל
+    ומושקעים מדי יום בשוקי ההון והכספים בארץ ובחו"ל.
+    <p>פרויקט פנסיה פתוחה, מבית <a href="http://www.hasadna.org.il/">הסדנא לידע ציבורי</a>, שם לו למטרה לחשוף את הדרך שבה כספי הפנסיה שלנו מנוהלים וכיצד הם מושקעים. באמצעות פנסיה פתוחה, תוכלו לגלות גם איך מושקעים כספי הפנסיה של הציבור כולו, וגם איך מושקעים כספי הפנסיה בקופה ספציפית (למשל, הקופה שלכם). תוכלו לענות על שאלות כמו: </p>
+    <ul>
+      <li>איזה אחוז מכספי הפנסיה שלנו מושקע בבנקים?</li>
```

Comment:
@eyalcohen4 Please style the `ul` from `list-style-type: disc` to `list-style-type: none`. And then and a class like:
```css
li:before {
  content: "-";
  /* And some padding i guessing */ 
}
```
In the end, should look like the `li` in: [link](https://drive.google.com/drive/u/0/folders/0B7RkqHKoREPqRnZiRnZoWFF1ek0)